src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -66,8 +66,8 @@ public static class ConfigurationBinder
         /// <param name="type">The type of the new instance to bind.</param>
         /// <returns>The new instance if successful, null otherwise.</returns>
         [RequiresUnreferencedCode(TrimmingWarningMessage)]
-        public static object Get(this IConfiguration configuration, Type type)
-            => configuration.Get(type, _ => { });
+        public static object? Get(this IConfiguration configuration, Type type)
+            => configuration.Get(type, null);
```

Comment:
We shouldn't be changing implementation during nullable changes. Can you undo this?